NGTrackForce/src/app/components/myinterview-view/myinterview-view.component.ts

```diff
@@ -149,24 +167,30 @@ export class MyInterviewComponent implements OnInit {
           .createInterview(this.newInterview, this.associate.id)
           .subscribe(res => {
             location.reload(false);
-          });
+          
+         });
       }
   }
+ 
 
-  updateInterview(interview: Interview) {
+  updateInterview(interview: Interview){
     if (!this.dateError){
         interview.isInterviewFlagged = +interview.isInterviewFlagged; // set it to number
-        interview.interviewDate = new Date(interview.interviewDate).getTime(); // convert into timestamp
+        interview.interviewDate = new Date(this.registerForm.value['dateinput']).getTime(); // convert into timestamp
         interview.dateSalesIssued = new Date(
```

Comment:
interview.dateSalesIssued and interview.dateAssociateIssued are never used. The lines that assign them can be deleted.